codegpt.el

```diff
@@ -139,17 +157,16 @@ that region in buffer."
    (read-string "Instruction: ")
    start end))
 
-(defconst codegpt-action-alist
-  `(("custom"  . "Write your own instruction")
-    ("doc"     . "Automatically write documentation for your code")
-    ("fix"     . "Find problems with it")
-    ("explain" . "Explain the selected code")
-    ("improve" . "Improve, refactor or optimize it"))
-  "Alist of code completion actions and its' description.")
+(defun codegept--execute-predefined-template (start end question)
+  "Ask predefined QUESTION for provided region.
+the START and END are boundaries of that region in buffer."
+  (codegpt--internal
+   question
+   start end))
```

Comment:
Why not use `codegpt--internal` directly?

Reply:
If somebody will want to extend this method it would be easier to add advice or override it, with codegpt--internal it would be more difficult, cause other methods also called it